src/loaders/common/loader_helper.py

```diff
@@ -289,6 +291,33 @@ def start_podman_service(uid: int):
     return proc
 
 
+def setup_callback_server_logs():
+    """Set up logs config file for the callback server"""
+    home = os.path.expanduser("~")
+    conf_path = os.path.join(home, CONTAINERS_CONF_PATH)
+    config = configparser.ConfigParser()
+    config.read(conf_path)
+    params = {
+        "seccomp_profile": "\"unconfined\"",
+        "log_driver": "\"k8s-file\""
+    }
+
+    modify = False
+    if not config.has_section("containers"):
+        config.add_section("containers")
+
+    for key, val in params.items():
+        if config.get("containers", key, fallback=None) != val:
+            config.set("containers", key, val)
+            if not modify:
+                modify = True
```

Comment:
```suggestion
            modify = True
```

Reply:
Not sure why need to keep assigning modify = True if already is

Reply:
It's a microoptimization that isn't worth the extra line of code

Reply:
... and come to think of it it might not even be an optimization. The only difference is that in one loop you assign the variable and in the other you check the value of the variable.

Reply:
Updated!

Reply:
moot now